doc-content/shared-kie-docs/src/main/asciidoc/KIE/Overview/installation-options-ref.adoc

```diff
@@ -0,0 +1,38 @@
+[id='installation-options-ref_{context}']
+= Installation environment options for {PRODUCT}
+
+With {PRODUCT}, you can set up a development environment to develop business applications, a runtime environment to run those applications to support decisions, or both.
+
+* *Development environment*: Typically consists of one {CENTRAL} installation and at least one {KIE_SERVER} installation. {CENTRAL} enables you to design decisions and other artifacts, and {KIE_SERVER} enables you to execute and test the artifacts that you created.
+
+* *Runtime environment*: Consists of one or more {KIE_SERVER} instances with or without {CENTRAL}. {CENTRAL} has an embedded {CONTROLLER}. If you install {CENTRAL}, use the *Menu* -> *Deploy* -> *Execution servers* page to create and maintain containers. If you want to automate {KIE_SERVER} management without {CENTRAL}, you can use the {HEADLESS_CONTROLLER}.
+
+You can also cluster both development and runtime environments. A clustered development or runtime environment consists of a unified group or "cluster" of two or more servers. The primary benefit of clustering {PRODUCT} development environments is high availability and enhanced collaboration, while the primary benefit of clustering {PRODUCT} runtime environments is high availability and load balancing. High availability decreases the chance of a loss of data when a single server fails. When a server fails, another server fills the gap by providing a copy of the data that was on the failed server. When the failed server comes online again, it resumes its place in the cluster. Load balancing shares the computing load across the nodes of the cluster to improve the overall performance.
+
+NOTE: Clustering of the runtime environment is currently supported on {EAP} {EAP_VERSION}
+ifdef::DM,PAM[]
+and {OPENSHIFT}
+endif::[]
+only. Clustering of {CENTRAL} is currently a Technology Preview feature that is not yet intended for production use.
+ifdef::DM,PAM[]
+For more information about Red Hat Technology Preview support, see https://access.redhat.com/support/offerings/techpreview/[Technology Preview Features Support Scope].
+endif::[]
+
+ifeval::["{context}" != "planning"]
+.Additional resources
+ifdef::DM,PAM[]
+* {URL_PLANNING_INSTALL}[_{PLANNING_INSTALL}_]
```

Comment:
This link in the generated doc points to a non-existing page. However I see it is supposed to link a 7.3 doc, so it would probably be released along this one. So take this just as a note to doublecheck, if it is like that.